bin/hermit.hcl

```diff
@@ -1,4 +1,4 @@
 env = {
-  "CLOUDBEAT_VERSION": "8.15.0",
+  "CLOUDBEAT_VERSION": "8.18.0",
```

Comment:
I might be mistaken here but I think the docker images of 8.18.0 might not be ready yet and since we use that in ci it might brake. If I remember correctly.